src/qibosoq/qick_programs.py

```diff
@@ -228,11 +242,11 @@ def acquire(
             debug (bool): if true prints the program actually executed
             average (bool): if true return averaged res, otherwise single shots
         """
-
+        if self.readouts_per_experiment is not None:
+            readouts_per_experiment = self.readouts_per_experiment
         # if there are no readouts, temporaray set 1 so that qick can execute properly
         reads_per_rep = 1 if readouts_per_experiment == 0 else readouts_per_experiment
```

Comment:
Here you can directly put
```py
if readouts_per_experiment == 0:
    return [], []
```
to avoid calling the qick `acquire` when `readouts_per_experiment == 0`.

Reply:
Actually I want to execute the program even if there are no readouts, to eventually just drive the qubit... Not sure if it is something useful